Build failure: devilutionx

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

1. Build `devilutionx`
2. Get error

### Can Hydra reproduce this build failure?

No, Hydra cannot reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/job/nixpkgs/trunk/devilutionx.x86_64-linux

### Relevant log output

```console
Running phase: unpackPhase
unpacking source archive /nix/store/vzfv28k585pym1xwi867xnk7grs5gqsf-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/share/doc/SDL_audiolib -DCMAKE_INSTALL_INFODIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include -DCMAKE_INSTALL_SBINDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/strip -DCMAKE_RANLIB=/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/ranlib -DCMAKE_AR=/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17 -DUSE_DEC_ADLMIDI:BOOL=FALSE -DUSE_DEC_BASSMIDI:BOOL=FALSE -DUSE_DEC_DRFLAC:BOOL=FALSE -DUSE_DEC_FLUIDSYNTH:BOOL=FALSE -DUSE_DEC_LIBOPUSFILE:BOOL=FALSE -DUSE_DEC_LIBVORBIS:BOOL=FALSE -DUSE_DEC_MODPLUG:BOOL=FALSE -DUSE_DEC_MPG123:BOOL=FALSE -DUSE_DEC_MUSEPACK:BOOL=FALSE -DUSE_DEC_OPENMPT:BOOL=FALSE -DUSE_DEC_SNDFILE:BOOL=FALSE -DUSE_DEC_WILDMIDI:BOOL=FALSE -DUSE_DEC_XMP:BOOL=FALSE -DUSE_RESAMP_SOXR:BOOL=FALSE -DUSE_RESAMP_SRC:BOOL=FALSE
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


copying path '/nix/store/xxd1yhb540fnx886qlmcfmjx9zrf21ak-smpq-1.6' from 'https://cache.nixos.org'...
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /nix/store/2ijmf0rg27gn78widiljfx9m7cfggjm0-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Checking for module 'flac'
--   Found flac, version 1.4.3
Package ogg was not found in the pkg-config search path.
Perhaps you should add the directory containing `ogg.pc'
to the PKG_CONFIG_PATH environment variable
Package 'ogg', required by 'flac', not found
Package ogg was not found in the pkg-config search path.
Perhaps you should add the directory containing `ogg.pc'
to the PKG_CONFIG_PATH environment variable
Package 'ogg', required by 'flac', not found
Package ogg was not found in the pkg-config search path.
Perhaps you should add the directory containing `ogg.pc'
to the PKG_CONFIG_PATH environment variable
Package 'ogg', required by 'flac', not found
Package ogg was not found in the pkg-config search path.
Perhaps you should add the directory containing `ogg.pc'
to the PKG_CONFIG_PATH environment variable
Package 'ogg', required by 'flac', not found
-- Module support is disabled.
-- Version: 8.1.1
-- Build type: Release
-- CXX_STANDARD: 20
-- Performing Test has_std_20_flag
-- Performing Test has_std_20_flag - Success
-- Performing Test has_std_2a_flag
-- Performing Test has_std_2a_flag - Success
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS - Success
-- Performing Test FMT_HAS_VARIANT
-- Performing Test FMT_HAS_VARIANT - Success
-- Required features: cxx_variadic_templates
-- Looking for strtod_l
-- Looking for strtod_l - not found
-- Performing Test HAVE_EXCEPTIONS
-- Performing Test HAVE_EXCEPTIONS - Success
-- Performing Test HAVE_STD_CLAMP
-- Performing Test HAVE_STD_CLAMP - Success
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Configuring done (1.3s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
build flags: -j32 SHELL=/nix/store/58br4vk3q5akf4g8lx0pqzfhn47k3j8d-bash-5.2p37/bin/bash
[  4%] Building CXX object 3rdparty/fmt/CMakeFiles/fmt.dir/src/format.cc.o
[  9%] Building CXX object 3rdparty/fmt/CMakeFiles/fmt.dir/src/os.cc.o
In file included from /build/source/3rdparty/fmt/include/fmt/format.h:48,
                 from /build/source/3rdparty/fmt/include/fmt/format-inl.h:29,
                 from /build/source/3rdparty/fmt/src/format.cc:8:
/build/source/3rdparty/fmt/include/fmt/core.h: In instantiation of 'constexpr fmt::v8::detail::value<Context> fmt::v8::detail::make_arg(T&&) [with bool IS_PACKED = true; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; type <anonymous> = fmt::v8::detail::type::string_type; T = fmt::v8::basic_string_view<char>&; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]':
/build/source/3rdparty/fmt/include/fmt/core.h:1855:77:   required from 'constexpr fmt::v8::format_arg_store<Context, Args>::format_arg_store(T&& ...) [with T = {fmt::v8::basic_string_view<char>&, const char (&)[3]}; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {fmt::v8::basic_string_view<char>, char [3]}]'
 1853 |         data_{detail::make_arg<
      |               ~~~~~~~~~~~~~~~~~
 1854 |             is_packed, Context,
      |             ~~~~~~~~~~~~~~~~~~~
 1855 |             detail::mapped_type_constant<remove_cvref_t<T>, Context>::value>(
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
 1856 |             std::forward<T>(args))...} {
      |             ~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:1872:38:   required from 'OutputIt fmt::v8::format_to(OutputIt, format_string<T ...>, T&& ...) [with OutputIt = appender; T = {basic_string_view<char>&, const char (&)[3]}; typename std::enable_if<detail::is_output_iterator<OutputIt, char>::value, int>::type <anonymous> = 0; format_string<T ...> = basic_format_string<char, basic_string_view<char>&, const char (&)[3]>]'
 1872 |   return {std::forward<Args>(args)...};
      |                                      ^
/build/source/3rdparty/fmt/include/fmt/format-inl.h:78:14:   required from here
   78 |     format_to(it, FMT_STRING("{}{}"), message, SEP);
      |     ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:3148:52:   in 'constexpr' expansion of 'fmt::v8::make_format_args<>(args#0, args#1)'
/build/source/3rdparty/fmt/include/fmt/core.h:1706:15: warning: possibly dangling reference to a temporary []
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |               ^~~
/build/source/3rdparty/fmt/include/fmt/core.h:1706:46: note: the temporary was destroyed at the end of the full expression 'fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >().fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >::map<fmt::v8::basic_string_view<char> >((*(const fmt::v8::basic_string_view<char>*)(& std::forward<fmt::v8::basic_string_view<char>&>((* & val)))))'
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
[ 13%] Linking CXX static library libfmt.a
[ 13%] Built target fmt
[ 18%] Building C object CMakeFiles/SDL_audiolib.dir/src/dr_mp3.c.o
[ 22%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/Decoder.cpp.o
[ 31%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/DecoderDrwav.cpp.o
[ 31%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/DecoderDrmp3.cpp.o
[ 36%] Building C object CMakeFiles/SDL_audiolib.dir/src/dr_wav.c.o
[ 40%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/DecoderFlac.cpp.o
[ 45%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/Resampler.cpp.o
[ 50%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/ResamplerSdl.cpp.o
[ 54%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/Processor.cpp.o
[ 59%] Building C object CMakeFiles/SDL_audiolib.dir/src/missing/sdl_load_file_rw.c.o
[ 63%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/SdlMutex.cpp.o
[ 68%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/aulib.cpp.o
[ 72%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/ResamplerSpeex.cpp.o
[ 81%] Building C object CMakeFiles/SDL_audiolib.dir/3rdparty/speex_resampler/resample.c.o
[ 81%] Building C object CMakeFiles/SDL_audiolib.dir/src/missing/sdl_rwsize.c.o
[ 86%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/stream_p.cpp.o
[ 90%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/sampleconv.cpp.o
[ 95%] Building CXX object CMakeFiles/SDL_audiolib.dir/src/Stream.cpp.o
In file included from /build/source/3rdparty/speex_resampler/resample.c:100:
/build/source/3rdparty/speex_resampler/resample_sse.h: In function 'inner_product_single':
/build/source/3rdparty/speex_resampler/resample_sse.h:45:14: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
   45 |    for (i=0;i<len;i+=8)
      |              ^
/build/source/3rdparty/speex_resampler/resample_sse.h: In function 'interpolate_product_single':
/build/source/3rdparty/speex_resampler/resample_sse.h:62:12: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
   62 |   for(i=0;i<len;i+=2)
      |            ^
/build/source/3rdparty/speex_resampler/resample_sse.h: In function 'inner_product_double':
/build/source/3rdparty/speex_resampler/resample_sse.h:84:14: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
   84 |    for (i=0;i<len;i+=8)
      |              ^
/build/source/3rdparty/speex_resampler/resample_sse.h: In function 'interpolate_product_double':
/build/source/3rdparty/speex_resampler/resample_sse.h:110:12: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
  110 |   for(i=0;i<len;i+=2)
      |            ^
/build/source/3rdparty/speex_resampler/resample.c: In function 'update_filter':
/build/source/3rdparty/speex_resampler/resample.c:674:20: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
  674 |          for (j=0;j<st->filt_len;j++)
      |                    ^
/build/source/3rdparty/speex_resampler/resample.c: In function 'SDL_audiolib_resampler_process_float':
/build/source/3rdparty/speex_resampler/resample.c:946:21: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
  946 |            for(j=0;j<ichunk;++j)
      |                     ^
/build/source/3rdparty/speex_resampler/resample.c:949:20: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
  949 |           for(j=0;j<ichunk;++j)
      |                    ^
/build/source/3rdparty/speex_resampler/resample.c: In function 'SDL_audiolib_resampler_process_int':
/build/source/3rdparty/speex_resampler/resample.c:1001:19: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
 1001 |          for(j=0;j<ichunk;++j)
      |                   ^
/build/source/3rdparty/speex_resampler/resample.c:1008:19: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
 1008 |          for(j=0;j<ichunk;++j)
      |                   ^
/build/source/3rdparty/speex_resampler/resample.c:1018:16: warning: comparison of integer expressions of different signedness: 'int' and 'unsigned int' []
 1018 |      for (j=0;j<ochunk+omagic;++j)
      |                ^
In file included from /build/source/src/dr_mp3.c:4:
In function 'drmp3_L3_decode_scalefactors',
    inlined from 'drmp3_L3_decode' at /build/source/3rdparty/dr_libs/dr_mp3.h:1785:9,
    inlined from 'drmp3dec_decode_frame' at /build/source/3rdparty/dr_libs/dr_mp3.h:2315:17:
/build/source/3rdparty/dr_libs/dr_mp3.h:1237:42: warning: writing 1 byte into a region of size 0 []
 1237 |             iscf[gr->n_long_sfb + i + 2] = (drmp3_uint8)(iscf[gr->n_long_sfb + i + 2] + (gr->subblock_gain[2] << sh));
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/dr_libs/dr_mp3.h: In function 'drmp3dec_decode_frame':
/build/source/3rdparty/dr_libs/dr_mp3.h:1202:30: note: at offset [41, 296] into destination object 'iscf' of size 40
 1202 |     drmp3_uint8 scf_size[4], iscf[40];
      |                              ^~~~
In file included from /build/source/src/aulib_log.h:4,
                 from /build/source/src/DecoderDrwav.cpp:6:
/build/source/3rdparty/fmt/include/fmt/core.h: In instantiation of 'constexpr fmt::v8::detail::value<Context> fmt::v8::detail::make_arg(T&&) [with bool IS_PACKED = true; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; type <anonymous> = fmt::v8::detail::type::string_type; T = std::__cxx11::basic_string<char>&; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]':
/build/source/3rdparty/fmt/include/fmt/core.h:1855:77:   required from 'constexpr fmt::v8::format_arg_store<Context, Args>::format_arg_store(T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&}; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}]'
 1853 |         data_{detail::make_arg<
      |               ~~~~~~~~~~~~~~~~~
 1854 |             is_packed, Context,
      |             ~~~~~~~~~~~~~~~~~~~
 1855 |             detail::mapped_type_constant<remove_cvref_t<T>, Context>::value>(
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
 1856 |             std::forward<T>(args))...} {
      |             ~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:1872:38:   required from 'void fmt::v8::print(FILE*, format_string<T ...>, T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; FILE = FILE; format_string<T ...> = basic_format_string<char, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >]'
 1872 |   return {std::forward<Args>(args)...};
      |                                      ^
/build/source/src/aulib_log.h:41:15:   required from 'void aulib::log::warnLn(fmt::v8::format_string<T ...>&&, Args&& ...) [with Args = {const char*}; fmt::v8::format_string<T ...> = fmt::v8::basic_format_string<char, const char*>]'
   41 |     fmt::print(stderr, "SDL_audiolib warning: {}\n",
      |     ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   42 |                fmt::format(std::forward<fmt::format_string<Args...>>(fmt_str),
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   43 |                            std::forward<Args>(args)...));
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/src/DecoderDrwav.cpp:35:27:   required from here
   35 |         aulib::log::warnLn("dr_wav: Cannot determine rwops size: {}",
      |         ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   36 |                            rwops_size == -1 ? "unknown error" : SDL_GetError());
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:3224:44:   in 'constexpr' expansion of 'fmt::v8::make_format_args<>((* & args#0))'
/build/source/3rdparty/fmt/include/fmt/core.h:1706:15: warning: possibly dangling reference to a temporary []
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |               ^~~
/build/source/3rdparty/fmt/include/fmt/core.h:1706:46: note: the temporary was destroyed at the end of the full expression 'fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >().fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >::map<std::__cxx11::basic_string<char> >((*(const std::__cxx11::basic_string<char>*)(& std::forward<__cxx11::basic_string<char>&>((* & val)))))'
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /build/source/src/aulib_log.h:4,
                 from /build/source/src/DecoderDrmp3.cpp:6:
/build/source/3rdparty/fmt/include/fmt/core.h: In instantiation of 'constexpr fmt::v8::detail::value<Context> fmt::v8::detail::make_arg(T&&) [with bool IS_PACKED = true; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; type <anonymous> = fmt::v8::detail::type::string_type; T = std::__cxx11::basic_string<char>&; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]':
/build/source/3rdparty/fmt/include/fmt/core.h:1855:77:   required from 'constexpr fmt::v8::format_arg_store<Context, Args>::format_arg_store(T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&}; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}]'
 1853 |         data_{detail::make_arg<
      |               ~~~~~~~~~~~~~~~~~
 1854 |             is_packed, Context,
      |             ~~~~~~~~~~~~~~~~~~~
 1855 |             detail::mapped_type_constant<remove_cvref_t<T>, Context>::value>(
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
 1856 |             std::forward<T>(args))...} {
      |             ~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:1872:38:   required from 'void fmt::v8::print(FILE*, format_string<T ...>, T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; FILE = FILE; format_string<T ...> = basic_format_string<char, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >]'
 1872 |   return {std::forward<Args>(args)...};
      |                                      ^
/build/source/src/aulib_log.h:41:15:   required from 'void aulib::log::warnLn(fmt::v8::format_string<T ...>&&, Args&& ...) [with Args = {const char*}; fmt::v8::format_string<T ...> = fmt::v8::basic_format_string<char, const char*>]'
   41 |     fmt::print(stderr, "SDL_audiolib warning: {}\n",
      |     ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   42 |                fmt::format(std::forward<fmt::format_string<Args...>>(fmt_str),
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   43 |                            std::forward<Args>(args)...));
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/src/DecoderDrmp3.cpp:35:27:   required from here
   35 |         aulib::log::warnLn("dr_mp3: Cannot determine rwops size: {}",
      |         ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   36 |                            rwops_size == -1 ? "unknown error" : SDL_GetError());
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:3224:44:   in 'constexpr' expansion of 'fmt::v8::make_format_args<>((* & args#0))'
/build/source/3rdparty/fmt/include/fmt/core.h:1706:15: warning: possibly dangling reference to a temporary []
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |               ^~~
/build/source/3rdparty/fmt/include/fmt/core.h:1706:46: note: the temporary was destroyed at the end of the full expression 'fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >().fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >::map<std::__cxx11::basic_string<char> >((*(const std::__cxx11::basic_string<char>*)(& std::forward<__cxx11::basic_string<char>&>((* & val)))))'
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /build/source/src/aulib_log.h:4,
                 from /build/source/src/DecoderFlac.cpp:3:
/build/source/3rdparty/fmt/include/fmt/core.h: In instantiation of 'constexpr fmt::v8::detail::value<Context> fmt::v8::detail::make_arg(T&&) [with bool IS_PACKED = true; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; type <anonymous> = fmt::v8::detail::type::string_type; T = std::__cxx11::basic_string<char>&; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]':
/build/source/3rdparty/fmt/include/fmt/core.h:1855:77:   required from 'constexpr fmt::v8::format_arg_store<Context, Args>::format_arg_store(T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&}; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}]'
 1853 |         data_{detail::make_arg<
      |               ~~~~~~~~~~~~~~~~~
 1854 |             is_packed, Context,
      |             ~~~~~~~~~~~~~~~~~~~
 1855 |             detail::mapped_type_constant<remove_cvref_t<T>, Context>::value>(
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
 1856 |             std::forward<T>(args))...} {
      |             ~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:1872:38:   required from 'void fmt::v8::print(FILE*, format_string<T ...>, T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; FILE = FILE; format_string<T ...> = basic_format_string<char, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >]'
 1872 |   return {std::forward<Args>(args)...};
      |                                      ^
/build/source/src/aulib_log.h:41:15:   required from 'void aulib::log::warnLn(fmt::v8::format_string<T ...>&&, Args&& ...) [with Args = {}; fmt::v8::format_string<T ...> = fmt::v8::basic_format_string<char>]'
   41 |     fmt::print(stderr, "SDL_audiolib warning: {}\n",
      |     ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   42 |                fmt::format(std::forward<fmt::format_string<Args...>>(fmt_str),
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   43 |                            std::forward<Args>(args)...));
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/src/DecoderFlac.cpp:45:27:   required from here
   45 |         aulib::log::warnLn("DecoderFlac: libFLAC requested read of zero bytes.");
      |         ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:3224:44:   in 'constexpr' expansion of 'fmt::v8::make_format_args<>((* & args#0))'
/build/source/3rdparty/fmt/include/fmt/core.h:1706:15: warning: possibly dangling reference to a temporary []
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |               ^~~
/build/source/3rdparty/fmt/include/fmt/core.h:1706:46: note: the temporary was destroyed at the end of the full expression 'fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >().fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >::map<std::__cxx11::basic_string<char> >((*(const std::__cxx11::basic_string<char>*)(& std::forward<__cxx11::basic_string<char>&>((* & val)))))'
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /build/source/src/aulib_log.h:4,
                 from /build/source/src/aulib.cpp:5:
/build/source/3rdparty/fmt/include/fmt/core.h: In instantiation of 'constexpr fmt::v8::detail::value<Context> fmt::v8::detail::make_arg(T&&) [with bool IS_PACKED = true; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; type <anonymous> = fmt::v8::detail::type::string_type; T = std::__cxx11::basic_string<char>&; typename std::enable_if<IS_PACKED, int>::type <anonymous> = 0]':
/build/source/3rdparty/fmt/include/fmt/core.h:1855:77:   required from 'constexpr fmt::v8::format_arg_store<Context, Args>::format_arg_store(T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&}; Context = fmt::v8::basic_format_context<fmt::v8::appender, char>; Args = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}]'
 1853 |         data_{detail::make_arg<
      |               ~~~~~~~~~~~~~~~~~
 1854 |             is_packed, Context,
      |             ~~~~~~~~~~~~~~~~~~~
 1855 |             detail::mapped_type_constant<remove_cvref_t<T>, Context>::value>(
      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
 1856 |             std::forward<T>(args))...} {
      |             ~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:1872:38:   required from 'void fmt::v8::print(FILE*, format_string<T ...>, T&& ...) [with T = {std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >}; FILE = FILE; format_string<T ...> = basic_format_string<char, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >]'
 1872 |   return {std::forward<Args>(args)...};
      |                                      ^
/build/source/src/aulib_log.h:41:15:   required from 'void aulib::log::warnLn(fmt::v8::format_string<T ...>&&, Args&& ...) [with Args = {short unsigned int&}; fmt::v8::format_string<T ...> = fmt::v8::basic_format_string<char, short unsigned int&>]'
   41 |     fmt::print(stderr, "SDL_audiolib warning: {}\n",
      |     ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   42 |                fmt::format(std::forward<fmt::format_string<Args...>>(fmt_str),
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   43 |                            std::forward<Args>(args)...));
      |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/src/aulib.cpp:114:27:   required from here
  114 |         aulib::log::warnLn("Unknown audio format spec: {}", Stream_priv::fAudioSpec.format);
      |         ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/3rdparty/fmt/include/fmt/core.h:3224:44:   in 'constexpr' expansion of 'fmt::v8::make_format_args<>((* & args#0))'
/build/source/3rdparty/fmt/include/fmt/core.h:1706:15: warning: possibly dangling reference to a temporary []
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |               ^~~
/build/source/3rdparty/fmt/include/fmt/core.h:1706:46: note: the temporary was destroyed at the end of the full expression 'fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >().fmt::v8::detail::arg_mapper<fmt::v8::basic_format_context<fmt::v8::appender, char> >::map<std::__cxx11::basic_string<char> >((*(const std::__cxx11::basic_string<char>*)(& std::forward<__cxx11::basic_string<char>&>((* & val)))))'
 1706 |   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
[100%] Linking CXX shared library libSDL_audiolib.so
[100%] Built target SDL_audiolib
Running phase: installPhase
install flags: -j32 SHELL=/nix/store/58br4vk3q5akf4g8lx0pqzfhn47k3j8d-bash-5.2p37/bin/bash install
[ 13%] Built target fmt
[100%] Built target SDL_audiolib
Install the project...
-- Install configuration: "Release"
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib/libSDL_audiolib.so.0.0.0
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib/libSDL_audiolib.so.0
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib/libSDL_audiolib.so
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/aulib.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/aulib_global.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/aulib_export.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/aulib_version.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/Decoder.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/Processor.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/Resampler.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/ResamplerSdl.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/ResamplerSpeex.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/Stream.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/DecoderDrmp3.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/DecoderDrwav.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/include/SDL_audiolib/Aulib/DecoderFlac.h
-- Installing: /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib/pkgconfig/SDL_audiolib.pc
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17
shrinking /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib/libSDL_audiolib.so.0.0.0
checking for references to /build/ in /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17...
patching script interpreter paths in /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17
stripping (with command strip and flags -S -p) in  /nix/store/pb4cz22gkfsz2qg4az8j0vd03ldq7625-SDL_audiolib-0-unstable-2022-04-17/lib
building '/nix/store/8s80gi06k5wryyq3g0vd1j5501l33h8d-devilutionx-1.5.3.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/h1sga0nkf89yz65qhy9027cp4kdvw1xm-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/share/doc/DevilutionX -DCMAKE_INSTALL_INFODIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/include -DCMAKE_INSTALL_SBINDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/strip -DCMAKE_RANLIB=/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/ranlib -DCMAKE_AR=/nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/3jv0qqmv8g4p6x9w7zlzndz52xmq75nw-devilutionx-1.5.3
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/8v6k283dpbc0qkdq81nb6mrxrgcb10i1-gcc-wrapper-14-20241116/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- 📚 zlib: dynamic system library
-- 📚 bzip2: dynamic system library
-- 📚 SDL2: dynamic system library
CMake Warning at CMake/finders/FindSDL2_image.cmake:1 (find_package):
  Found package configuration file:

    /nix/store/3f3nw92410jgg3smyq7ihl5pj8rdhymv-SDL2_image-2.8.5/lib/cmake/SDL2_image/sdl2_image-config.cmake

  but it set SDL2_image_FOUND to FALSE so package "SDL2_image" is considered
  to be NOT FOUND.  Reason given by package:

  SDL2_image could not be found because dependency webp could not be found.

Call Stack (most recent call first):
  CMake/Dependencies.cmake:95 (find_package)
  CMake/Dependencies.cmake:105 (_find_SDL_image)
  CMakeLists.txt:308 (include)


-- Found PkgConfig: /nix/store/2ijmf0rg27gn78widiljfx9m7cfggjm0-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
Package libtiff-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtiff-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtiff-4', required by 'SDL2_image', not found
Package libtiff-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtiff-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtiff-4', required by 'SDL2_image', not found
Package libtiff-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtiff-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtiff-4', required by 'SDL2_image', not found
Package libtiff-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libtiff-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libtiff-4', required by 'SDL2_image', not found
-- Found SDL_image
-- 📚 SDL_image: dynamic system library
CMake Error at /nix/store/ciiwq1ymzac9m5azhf4higkynmy3f8f7-cmake-3.31.5/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find webp (missing: webp_LIBRARY webp_INCLUDE_PATH
  webpdemux_LIBRARY webpdemux_INCLUDE_PATH)
Call Stack (most recent call first):
  /nix/store/ciiwq1ymzac9m5azhf4higkynmy3f8f7-cmake-3.31.5/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/3f3nw92410jgg3smyq7ihl5pj8rdhymv-SDL2_image-2.8.5/lib/cmake/SDL2_image/Findwebp.cmake:31 (find_package_handle_standard_args)
  /nix/store/ciiwq1ymzac9m5azhf4higkynmy3f8f7-cmake-3.31.5/share/cmake-3.31/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  /nix/store/3f3nw92410jgg3smyq7ihl5pj8rdhymv-SDL2_image-2.8.5/lib/cmake/SDL2_image/sdl2_image-config.cmake:57 (find_dependency)
  CMake/finders/FindSDL2_image.cmake:1 (find_package)
  CMake/Dependencies.cmake:95 (find_package)
  CMake/Dependencies.cmake:115 (_find_SDL_image)
  CMakeLists.txt:308 (include)


-- Configuring incomplete, errors occurred!
```

### Additional context

[Version 1.5.4 is also out](https://github.com/diasurgical/DevilutionX/releases/tag/1.5.4)

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.14.2, NixOS, 25.05 (Warbler), 25.05.20250412.2631b0b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.1`
 - nixpkgs: `/nix/store/5ds20jm3x2s4z7wn3581r6lc9ybmh45b-source`

### Notify maintainers

@karolchmist
@aanderse 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/games/devilutionx/default.nix
+++ b/pkgs/games/devilutionx/default.nix
@@ -14,6 +14,8 @@
   flac,
   fmt,
   libpng,
+  libtiff,
+  libwebp,
   smpq,
 }:
 
@@ -98,6 +100,8 @@ stdenv.mkDerivation rec {
     bzip2
     fmt
     libpng
+    libtiff
+    libwebp
     libsodium
     SDL2_classic
     SDL2_image

